spring-integration-graphql/src/main/java/org/springframework/integration/graphql/outbound/GraphQlMessageHandler.java

```diff
@@ -143,11 +143,11 @@ protected Object handleRequestMessage(Message<?> requestMessage) {
 		}
 		else  {
 			Assert.notNull(this.operationExpression, "'operationExpression' must not be null");
-			Assert.notNull(this.idExpression, "'idExpression' must not be null");
+			Assert.notNull(this.executionIdExpression, "'executionIdExpression' must not be null");
```

Comment:
We don't need this because it is not null by default and null cannot pass the setter.